### PR TITLE
Fix create test shop feedback

### PR DESCRIPTION
**Pull request by unabris**
I have created a new file called test-shop-versions.yml under _data where put the available ePages versions. It consists in:
- **key**: identifier of the version.
- **flag**: the value of the _ShopTypeAlias_ input on the form.
- **text**: the text that is going to appear after clicking in the _"Join now"_ button.

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -73,32 +80,48 @@
...
-  <a data-toggle="modal" href="#modal-popup" class="cta">Join now</a>
+  <div class="row">
+    <div class="col-md-6">
+      <h2>Create Apps for ePages Now!</h2>
+      <p>ePages Now is our new product generation that let's you develop awesome storefront apps such as chat tools or exit popup plugins.</p>
+      <a data-toggle="modal" href="#modal-popup-now" class="cta">Join now</a>
+    </div>
+    <div class="col-md-6">
+      <h2>Create Apps for ePages 6!</h2>
+      <p>ePages 6 is our current full-blown shop solution that enables yout o get many different types of data for your service.</p>
```
Comment (BirgitBader) [inline pages/index.html]: Please fix this typo: 

... that enables you to get many different...

File: pages/index.html [hunk 2]
```diff
@@ -73,32 +80,48 @@
...
+    <h2>More resources</h2>
+    <div class="col-sm-8  col-sm-offset-2  col-md-4  col-md-offset-0">
       <h4><span aria-hidden="true" class="fa-cogs"></span>Developer Blog</h4>
-      <p>Read about the latest developments at ePages and current industry trends at <a href="{{ '/blog/index.html' | prepend: site.baseurl }}">our blog.</a></p>
+      <p>Read about the latest developments at ePages and current industry trends at <a href="{{ '/blog/index.html' | prepend: site.baseurl }}"><span style="white-space: nowrap">our blog</span>.</a></p>
     </div>
-    <div>
+    <div class="col-sm-8  col-sm-offset-2  col-md-4  col-md-offset-0">
       <h4><span aria-hidden="true" class="fa-pencil"></span>Open Source Projects</h4>
       <p>We have various open source projects which you can use and contribute to.</p>
```
Comment (BirgitBader) [inline pages/index.html]: Please change to:

Our development teams benefit a lot from open-source software. Here's a list of our [open-source projects](https://github.com/ePages-de) that you can use and contribute to.

**Comment by BirgitBader** [comment]
@unabris @Gristof Just some small comments/amendments - other than that, I'm fine :)

File: pages/index.html [hunk 3]
```diff
@@ -73,32 +80,48 @@
...
 </section>
 
 <!--  REGISTER-->
 
 <section id="register" class="container-fluid">
-  <h2>Not registered for the developer program yet?</h2>
-  <a data-toggle="modal" href="#modal-popup" class="cta">Join now</a>
+  <div class="row">
+    <div class="col-md-6">
+      <h2>Create Apps for ePages Now!</h2>
```
Comment (BirgitBader) [inline pages/index.html]: Create apps for ePages Now!

File: pages/index.html [hunk 4]
```diff
@@ -73,32 +80,48 @@
...
+      <p>ePages Now is our new product generation that let's you develop awesome storefront apps such as chat tools or exit popup plugins.</p>
+      <a data-toggle="modal" href="#modal-popup-now" class="cta">Join now</a>
+    </div>
+    <div class="col-md-6">
+      <h2>Create Apps for ePages 6!</h2>
```
Comment (BirgitBader) [inline pages/index.html]: Create apps for ePages 6!

File: pages/index.html (hunk 2 above)
Comment (unabris) [inline pages/index.html]: I have changed the text and the link is opening on a new tab

File: pages/index.html (hunk 1 above)
Comment (unabris) [inline pages/index.html]: Done

File: pages/index.html (hunk 4 above)
Comment (unabris) [inline pages/index.html]: Done

File: pages/index.html (hunk 3 above)
Comment (unabris) [inline pages/index.html]: Done

**Comment by unabris** [comment]
I have implemented a JavaScript function that scrolls to the register when you click on _"Sign up for free"_ and also another function that resets the registration forms after submit, so you can create another shop clicking again in _"Join Now"_

**Pull request merged**